src/views/List.jsx

```diff
@@ -1,15 +1,44 @@
+import { useState } from 'react';
+
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [searchTerm, setSearchTerm] = useState('');
+
+	const clearSearchTerm = () => {
+		setSearchTerm('');
+	};
+
 	return (
 		<>
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
+			<form onSubmit={(e) => e.preventDefault()}>
+				<label>
+					Filter Items
+					<input
+						type="text"
+						placeholder="start typing here..."
+						id="filter"
+						name="filter"
+						value={searchTerm}
+						onChange={(e) => {
+							setSearchTerm(e.target.value);
+						}}
+					/>
+				</label>
+			</form>
+			<button onClick={clearSearchTerm}>clear</button>
```

Comment:
Nice job putting the <button> element outside of the form so the button is not associated with the input field (when user presses enter, the input won't be cleared)

I would add type="button" for better semantics and accessibility

Reply:
Great point! Updated to include for better accessibility.